server/src/main/java/fr/uga/l3miage/tp4/models/EcosSessionProgrammationStepEntity.java

```diff
@@ -0,0 +1,22 @@
+package fr.uga.l3miage.tp4.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import java.time.LocalDateTime;
+
+@Entity
+public class EcosSessionProgrammationStepEntity {
+    @Id
+    private Long id;
+    private final LocalDateTime dateTime = LocalDateTime.now();
```

Comment:
non ici le. final dans l'uml, voulais dire qu'il ne peux pas être mis à jour donc
```suggestion
    @Column(updatable = false)
    private LocalDateTime dateTime;
```